opentracing-impl-java8/src/main/java/io/opentracing/AbstractSpan.java

```diff
@@ -23,14 +23,14 @@
 
 abstract class AbstractSpan implements Span, SpanContext {
 
-    final String operationName;
+    protected final String operationName;
 
     protected final Map<String,String> baggage = new HashMap<>();
```

Comment:
Could you add to this PR making this private? Since all access to it is possible (and safer) through provided methods.